Source/Modules/Documentarian/Documentation/reference/cmdlets/_index.md

```diff
@@ -2,8 +2,25 @@
 title: Cmdlet Reference
 linkTitle: Cmdlets
 summary: Reference documentation for the PowerShell cmdlets in the **Documentarian** module.
-description: >-
-  Reference documentation for the PowerShell cmdlets in the **Documentarian** module.
+description: Reference documentation for the PowerShell cmdlets in the **Documentarian** module.
+no_list: true
+
+Module Name: Documentarian
+Module Guid: 2422ec90-815c-4c1d-8ec1-4c9b082f2909
+Download Help Link: https://microsoft.github.io/Documentarian/modules/documentarian/help/
+Help Version: 0.0.1
+Locale: en-US
 ---
 
-Not yet written.
+The **Documentarian** module includes commands for inspecting, editing, and maintaining Markdown
+documents.
```

Comment:
I updated these values, but I also removed the module name header. I think we'll need to have a bit of noodling about this for the export-to-maml problem, _but_ we should approach that holistically.

The alternative is to leave the `_index.md` file without this information and keep the `Documentarian.md` file but marked as draft so Hugo never builds it.